src/main/java/com/google/sps/data/Student.java

```diff
@@ -61,4 +61,17 @@ public void addToScheduledSessions(TutorSession tutoringSession) {
         this.scheduledSessions = scheduledSessionsList.toArray(new TutorSession[0]);
     }
 
+    /** Marks the tutoring session that has the given tutor's email as rated.*/
+    public void markTutoringSessionAsRatedByTutorEmail(String tutorEmail, int newRating) {
+        List<TutorSession> scheduledSessionsList = new LinkedList<TutorSession>(Arrays.asList(this.scheduledSessions));
```

Comment:
I think it might be better if we made the learning and scheduledSessions array an ArrayList instead so we don't have to convert every time. I think this would also help with datastore because we can store Lists but not arrays in the entities.

Reply:
I'll actually make a PR for this and change the arrays to ArrayLists.

Reply:
That would be great! Thank you for doing that